test/spec/Feature/Query/PlanSpec.hs

```diff
@@ -194,6 +194,62 @@ spec actualPgVersion = do
         resStatus `shouldBe` Status { statusCode = 200, statusMessage="OK" }
         totalCost `shouldBe` 1.29
 
+    it "outputs the total cost for 2 upserts" $ do
+      r <- request methodPost "/tiobe_pls"
+            [("Prefer","handling=strict; resolution=merge-duplicates"), ("Accept","application/vnd.pgrst.plan+json")]
```

Comment:
There seems to be a typo here. It should be `,` (multiple headers) instead of `;` (for parameters). The resolution is not being applied here.

Reply:
I think we don't need the `handling=strict` for this case too